subsys/bluetooth/services/ots/ots.c

```diff
@@ -184,6 +232,11 @@ int bt_ots_obj_add(struct bt_ots *ots,
 		return -EINVAL;
 	}
 
+	CHECKIF(!ots_obj_validate_prop_against_oacp(obj_init->props, ots->features.oacp)) {
+		LOG_DBG("Object properties are not a subset of OACP");
```

Comment:
Might be worth logging the values of obj_init->props and ots->features.oacp, so it will possible to debug.